modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -62,9 +62,11 @@ module.exports = ({
       ]
     },
     resolve: {
-      extensions: [ '*', '.js', '.vue', '.json' ],
+      extensions: [ '.*', '.js', '.vue', '.json' ],
       alias: {
-        vue$: 'vue/dist/vue.runtime.esm.js',
+        /* vue$: '../../../../../../node_modules/@vue/runtime-dom', */
+        vue$: '@vue/runtime-dom',
```

Comment:
Hang on... are we not using compatibility mode? I'm not asking for you to change back at this late date because it looks like it worked out well, but am I wrong when I write in the migration guide about the benefits of compatibility mode?

Reply:
Right, sorry for the late reply lol, removed all occurences.